src/training/train.py

```diff
@@ -40,7 +42,7 @@ def create_folds(values: list, n_folds: int, seed: int) -> dict:
 def train_imbalanced_model(
     X_train: pd.DataFrame,
     y_train: pd.Series,
-    model: ClassifierMixin,
```

Comment:
Somehow ClassifierMixin does not work nicely with mlflow. This does not has to be necessary